### ISSUE TITLE
GWT Validator Warning

### ISSUE THREAD

**Issue by Nicolas-M** [Bathlamos/RTDC#67]
The build is failing, and I suspect there's an issue with GWT and Validators (see below). @Bathlamos, could you take a look when you have a chance? Thanks!
```
:web:compileGwtJul 22, 2015 3:20:51 AM java.util.prefs.FileSystemPreferences$1 run
INFO: Created user preferences directory.
Compiling module rtdc.web.rtdc-web
   Computing all possible rebind results for 'com.google.gwt.useragent.client.UserAgentAsserter'
      Rebinding com.google.gwt.useragent.client.UserAgentAsserter
         Checking rule <generate-with class='com.google.gwt.validation.rebind.ValidatorGenerator'/>
            [WARN] Detected warnings related to 'javax.validation.Validator'.   Are validation-api-<version>.jar and validation-api-<version>-sources.jar on the classpath?
            [WARN] Unknown type 'javax.validation.Validator' specified in deferred binding rule
:web:compileGwt FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':web:compileGwt'.
> Process 'command '/usr/lib/jvm/java-7-oracle/bin/java'' finished with non-zero exit value 137
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
BUILD FAILED
```

**Comment by Nicolas-M** [comment]
I fixed the build with commit ab38778b1beab19d0117e377fdbb5a75519700ec. I thought Travis had a problem with GWT, but it seems that it didn't like one of the product flavors I had added (See #66).

Should we still solve this warning, or is it safe to ignore it? I'll wait on your answer before closing this issue.

**Comment by Nicolas-M** [comment]
The build seems to fail 2 times out of 3, with the GWT compile marked as the cause of the failure. I think we should investigate this. It's not too urgent, but this should be resolved soon.
